maplibre/Cargo.toml

```diff
@@ -69,4 +69,4 @@ csscolorparser = { version = "0.5", features = ["serde", "cint"]}
 cint = "0.2"
 
 [build-dependencies]
-maplibre-build-tools = { path = "../maplibre-build-tools", version = "0.1.0" }
+maplibre-build-tools = { path = "../maplibre-build-tools", version = "0.1.0", features = ["sqlite"] }
```

Comment:
I think we want to add a `embed-static-tiles` feature flag to `maplibre`.
That feature flag will enable the `sqlite` feature of the build-tools and conditionally embed tiles
 in the `build.rs` file.
Such that one can conditionally compile `maplibre`, once with sqlite support during building and once without.